manifests/testing/kubevirt-testing-infra.yaml

```diff
@@ -50,7 +50,7 @@ spec:
       serviceAccountName: kubevirt-testing
       containers:
         - name: target
-          image: quay.io/kubevirt/disks-images-provider:v1.4.0
+          image: quay.io/kubevirt/disks-images-provider:v1.6.0-beta.0
```

Comment:
Can this image be used if older releases? (i.e. 4.19/18/17/16)

Reply:
I don't know but master branch from this repo shouldn't be used to run those versions AFAIK.